general/tools.md

```diff
@@ -3,20 +3,26 @@
 At source{d} we try to use tools that are as simple as possible. If you can think of any tools that are simpler, better or cheaper please let us know! Here we will give instructions on how to make the best use of the tools we currently use.
 
 ## General
+
 These are the tools that every sourcerer uses:
 * [Github](https://github.com)
 * [Slack](https://slack.com)
-* [Gmail](https://gmail.com)
-  * There is no internal email at source{d}, we use GitHub for all async communications
-* [Zoom](http://zoom.us/)
-* [Hangouts Meet](http://hangouts.google.com)
+* [Gmail](https://gmail.com) (internal email usage is minimal at source{d}, most communication happens on Slack or GitHub)
+* [KinHR](https://kinhr.com)
 * [Google Calendar](http://calendar.google.com)
-* [Calendly](https://calendly.com)
 * [Google Docs](http://docs.google.com)
-* [Perdoo](https://www.perdoo.com)
+* [Donut](https://src-d.slack.com/apps/A11MJ51SR-donut?page=1)
```

Comment:
```suggestion
* [Donut](https://src-d.slack.com/apps/A11MJ51SR-donut?page=1) Slack app that will randomly match you up with another mate to have an informal conversation from time to time to know each other better ([see more](https://github.com/src-d/guide/blob/master/communication/donut.md.)
```

* [Donut](https://src-d.slack.com/apps/A11MJ51SR-donut?page=1) Slack app that will randomly match you up with another mate to have an informal conversation from time to time to know each other better ([see more](https://github.com/src-d/guide/blob/master/communication/donut.md))